static/captivate/js/main.js

```diff
@@ -40,10 +40,9 @@
     var customEllipsis = function () {
         var els = '.latest-video-title';
         var maxHeight = parseInt($(els).css('max-height')) + 1;
-        var lineHeight = isLoadMore ? 0 : parseInt($(els).css('line-height'));        
```

Comment:
It's not relevant but this was raised by scrutinizer check and decided to include the fix here. Should I revert this fix?